core/src/main/java/org/infinispan/globalstate/GlobalConfigurationManager.java

```diff
@@ -24,20 +24,38 @@
     * @param configuration the configuration object
     * @param flags the flags to apply
     */
-   Configuration createCache(String cacheName, Configuration configuration, EnumSet<CacheContainerAdmin.AdminFlag> flags);
+   Configuration createCache(String cacheName, Configuration configuration, EnumSet<AdminFlag> flags);
```

Comment:
to think (in another PR): the return values aren't used. probably they should return void.

Reply:
Yes, I'm trying to remember why I did this, and I think it's connected to the persistence of configuration into the server model, but I need to fetch that from the back of my brain.

Reply:
no worries :+1: